housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/validate/UnexpectedValidationFailure.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.housekeeping.tool.vacuum.validate;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.io.Writer;
+
+import com.hotels.housekeeping.tool.vacuum.conf.Table;
+
+class UnexpectedValidationFailure implements ValidationFailure {
+
+  private final Table table;
+  private final Exception e;
+
+  UnexpectedValidationFailure(Table table, Exception e) {
+    this.table = table;
+    this.e = e;
+  }
+
+  @Override
+  public String getMessage() {
+    Writer stringWriter = new StringWriter();
+    try (PrintWriter writer = new PrintWriter(stringWriter)) {
```

Comment:
Could also use commons-lang `ExceptionUtils` to turn this into a one-liner? But then it means adding another dependency...

Reply:
ah didn't even look for anything, I'll change it if we got the dependency already

Reply:
this will go with as it is now a MissingTableValidationFailure